Assets/HoloToolkit/UX/Scripts/Lines/Bezeir.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+
+namespace MRTK.UX
+{
+    public class Bezeir : LineBase
```

Comment:
Should this be [`Bezier`](https://en.wikipedia.org/wiki/B%C3%A9zier_curve)?

Reply:
D'oh. I always forget the 'I before E except after B' rule.